charts/govuk-apps-conf/rules/router/alerts.yaml

```diff
@@ -0,0 +1,11 @@
+groups:
+- name: RouterAlerts
+  rules:
+  - alert: RouterBackendHandlerResponseErrorRate
+    expr: sum(rate(router_backend_handler_response_duration_seconds_count{response_code=~"^5..", job="apps/router"}[5m])) / sum(rate(router_backend_handler_response_duration_seconds_count{job="apps/router"}[5m])) > 0.1
```

Comment:
I know it is a test but it seems it only considers 5xx HTTP codes as error, might need to expand that to include 4xx also?

Reply:
So in [ec2 production grafana](https://grafana.blue.staging.govuk.digital/dashboard/file/2ndline_health.json?refresh=1m&orgId=1), we care about 4xx (excluding 429-too many requests), 5xx and 429 as separate. We have a graph for them each.





[4:58](https://gds.slack.com/archives/C013F737737/p1655999882630559)
I was wondering about combining the 3 so that we have one alert. Obviously the alert will trigger more often as it is the sum…